compiler/test/dotc/comptest.scala

```diff
@@ -1,9 +1,14 @@
 package dotc
 
-import dotty.tools.dotc.ParallelTesting
+import dotty.tools.dotc.vulpix.ParallelTesting
+
+import scala.concurrent.duration._
 
 object comptest extends ParallelTesting {
 
+  def maxDuration = 3.seconds
+  def numberOfSlaves = 5
+  def safeMode = false
```

Comment:
This is a legacy main used by some folks in the lab - it's only running a single test. No need to restart VMs after this single test.